packages/node_modules/@ciscospark/spark-core/src/lib/credentials/credentials.js

```diff
@@ -341,6 +371,10 @@ const Credentials = SparkPlugin.extend({
     const supertoken = this.supertoken;
     const tokens = this.userTokens.models;
 
+    // clear refresh timer
+    if (this.refreshTimer) {
+      clearTimeout(this.refreshTimer);
```

Comment:
I think you also want to unset the refresh time when you clear it, right? Otherwise `this.refreshTimer` would be truthy at unexpected times.